src/Nethermind/Nethermind.Evm/Precompiles/IdentityPrecompile.cs

```diff
@@ -29,7 +29,7 @@ public long DataGasCost(in ReadOnlyMemory<byte> inputData, IReleaseSpec releaseS
 
         public (ReadOnlyMemory<byte>, bool) Run(in ReadOnlyMemory<byte> inputData, IReleaseSpec releaseSpec)
         {
-            return (inputData, true);
+            return (inputData.ToArray(), true);
```

Comment:
Not enturely sure why, but IdentityPrecompile isn't really used so shouldn't really matter